.codeclimate.yml

```diff
@@ -1,6 +1,9 @@
 exclude_paths:
- - "/lib/testing.php"
- - "/vendor/"
+ - "/examples/*"
 engines:
   phpcodesniffer:
     enabled: true
+    config:
+      - custom_exclude_paths: "*/test.php","/vendor/"
```

Comment:
Yes - This allows you to exclude a file specifically from PHP_CodeSniffer but not from other analysis.